### PR TITLE
Create use-Runtime-Server-migrationservice.json

### PR DESCRIPTION
**Pull request by apduvuri**
ARM Template for runtime server in migration service for Azure Database for PostgreSQL

### CODE REVIEW

**Comment by apduvuri** [comment]
#sign-off

File: postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            }
+        },
+        "location": {
+            "defaultValue": "[resourceGroup().location]",
+            "type": "String",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+         "RuntimeServer": {
```
Comment (ambrahma) [inline postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json:74]: migrationRuntimeServerResourceId

File: postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+          "type": "string",
```
Comment (ambrahma) [inline postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json:75]: Add 
            "defaultValue": ""

File: postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            "metadata": {
+                "description": "String array containing the names of databases to be migrated from source server to target server."
+            }
+        },
+		"migrationOption": {
+			"defaultValue": "ValidateAndMigrate",
+            "allowedValues": [
+                "Validate",
+                "Migrate",
+				"ValidateAndMigrate"
```
Comment (ambrahma) [inline postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json:92]: align values

File: postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                    },
+                    "sourceServerUsername": "[parameters('sourceServerLoginName')]",
+                    "targetServerUsername": "[parameters('targetServerLoginName')]"
+                },
+                "migrationMode": "[parameters('migrationMode')]",
+                "overwriteDbsInTarget": "[parameters('overwriteDbsInTarget')]",
+				"migrationOption": "[parameters('migrationOption')]",
+                "sslMode": "[parameters('SslMode')]",
+                "sourceType": "[parameters('SourceType')]",
+                "migrationInstanceResourceId": "[parameters('RuntimeServer')]"
```
Comment (ambrahma) [inline postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json]: "migrationInstanceResourceId": "[if(empty(parameters('migrationRuntimeServerResourceId')), json('null'), parameters('migrationRuntimeServerResourceId'))]",

File: postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json (hunk 3 above)
Comment (apduvuri) [reply postgresql-flexible-server/Migration/ARM-Template/use-Runtime-Server-migrationservice.json:93]: Completed

**Review by ambrahma — APPROVED** [review]

**Pull request merged**